packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapHeatmapController.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+// Defines heatmap UI options writable from Flutter.
+@protocol FLTGoogleMapHeatmapOptionsSink
```

Comment:
We don't need this Sink protocol. We can just add all the methods to `FLTHeatmapsController`. It reduces a lot of complication of the code.

Reply:
Done. I also attempted to mimic the other changes from your refactor. Let me know if you have any comments on those changes.